docs/markdown/30-testing/01-others.md

```diff
@@ -13,4 +13,88 @@ If both cases, you can use **bats** to help you write a test suite for your acti
 
 # Testing Shell script with Bats
 
-TODO example
+## Test folder
+
+```yaml
+.github/
+    actions/
+        runs-using-docker/
+            cleanup.sh
+            main.sh
+            ...
```

Comment:
This refers to the implementation inside the school repository.
For the slide, can we use an fake action like

```suggestion
action.yaml
src/
    cleanup.sh
    main.sh
    setup.sh
```

Reply:
Done